### PR TITLE
changed single-letter param names and resolved some todo's that were …

### PR DESCRIPTION
**Pull request by joaoritter**
…noted

### CODE REVIEW

**Review by tauren — APPROVED** [review]
A few comments/questions, otherwise looks good!

File: contracts/FountainV1.sol [hunk 1]
```diff
@@ -74,98 +81,161 @@ contract FountainV1 {
...
     // The total number of MoneyPools created, which is used for issuing MoneyPool IDs.
     // MoneyPools should have an id > 0, 0 should not be a moneyPoolId.
     uint256 public moneyPoolCount;
 
     // The contract currently only supports sustainments in DAI.
-    IERC20 public DAI;
+    address public DAI;
 
-    event MoneyPoolCreated(
+    event CreateMoneyPool(
```
[the remark this answers is not in the document]
Comment (joaoritter) [reply contracts/FountainV1.sol:96]: This is reflecting on the Uniswap contract and how they do it. In fact, they should omit the noun altogether in favor of "Create", "Update", etc

File: contracts/FountainV1.sol [hunk 2]
```diff
@@ -177,34 +247,22 @@ contract FountainV1 {
...
         // Otherwise set to the portion of the amount it'll take for sustainability to be reached
-        uint256 sustainabilityAmount = currentMoneyPool.currentSustainment.add(
-            a
-        ) <= currentMoneyPool.sustainabilityTarget
-            ? a
-            : currentMoneyPool.currentSustainment >=
-                currentMoneyPool.sustainabilityTarget
-            ? 0
-            : currentMoneyPool.sustainabilityTarget.sub(
+        uint256 sustainabilityAmount;
```
[the remark this answers is not in the document]
Comment (joaoritter) [reply contracts/FountainV1.sol:250]: yep!

[the remark this answers is not in the document]
Comment (joaoritter) [reply contracts/FountainV1.sol:250]: yep!

File: contracts/FountainV1.sol [hunk 3]
```diff
@@ -74,98 +81,161 @@ contract FountainV1 {
...
+        );
+
+        return true;
     }
 
     /// @notice Contribute a specified amount to the sustainability of the specified address's active MoneyPool.
     /// @notice If the amount results in surplus, redistribute the surplus proportionally to sustainers of the MoneyPool.
-    /// @param w Address to sustain.
-    /// @param a Amount of sustainment.
-    function sustain(address w, uint256 a) external payable {
```
[the remark this answers is not in the document]
Comment (joaoritter) [reply contracts/FountainV1.sol:160]: ah yes, this is something i meant to bring up. Payable is only necessary for inbound/outbound ETH transfers. ERC-20 transfers don't need it.

**Pull request merged**